app/views/spree/admin/orders/customer_details/edit.html.haml

```diff
@@ -1,4 +1,4 @@
-= render :partial => 'spree/admin/shared/order_tabs', :locals => { :current => 'Customer Details' }
+= render :partial => 'spree/admin/shared/order_page_title', :locals => { :current => 'Customer Details' }
```

Comment:
I think you need to render the order tabs here. otherwise the customer details page will not have the tabs visible, rigth?

Reply:
added order_tabs in and removed rocket